randovania/games/prime2/logic_database/Agon Wastes.txt

```diff
@@ -1586,25 +1586,23 @@ Extra - in_dark_aether: False
   * Extra - dock_name: West
   > Door to Command Center Access (Top)
       All of the following:
-          After Central Mining Station Pirates
+          Morph Ball and After Central Mining Station Pirates
           Any of the following:
-              All of the following:
-                  Morph Ball
-                  Any of the following:
-                      Spider Ball
-                      Morph Ball Bomb and Space Jump Boots and Bomb Space Jump (Intermediate)
-              # https://www.youtube.com/watch?v=mprL_DAECUI
-              Standable Terrain (Beginner) and Use Screw Attack (Space Jump)
-              All of the following:
-                  Space Jump Boots
-                  Any of the following:
-                      # https://youtu.be/qvaGUSk41g8?list=PLx6vT1x-80zuA1vUepQUSqYOQeHiRNbt5&t=22 Slope Jump
-                      Slope Jump (Expert)
-                      Scan Visor and Combat/Scan Dash (Expert) and Standable Terrain (Expert)
+              Spider Ball
+              Morph Ball Bomb and Space Jump Boots and Bomb Space Jump (Intermediate)
   > Door to Command Center Access (Bottom)
       After Central Mining Station Pirates
   > Event - Central Mining Station Pirates
       Trivial
+  > Center Platforms
+      All of the following:
+          Space Jump Boots and After Central Mining Station Pirates
+          Any of the following:
+              # Slope Jump: https://www.youtube.com/watch?v=qJQwGw-SuQI
+              Slope Jump (Expert)
+              Scan Visor and Combat/Scan Dash (Expert) and Standable Terrain (Beginner)
```

Comment:
not an echoes db maintainer, but what i personally like to do for the databases I maintain is to have a `TODO: add video` comment if we don't have a video yet.

Reply:
ironically this will be counterproductive with #6365 merged

Reply:
add to the bucket list of new rules to add to that feature

Reply:
oh right i forgot about that lol